ui/src/components/ModelCarousel.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import Component from 'react-component-component';
+import { Link } from 'react-router-dom';
+import Spinner from 'react-spinkit';
+import { stringify } from 'query-string';
+
+import { Row } from 'theme/system';
+import ArrowLeftIcon from 'icons/ArrowLeftIcon';
+import ArrowRightIcon from 'icons/ArrowRightIcon';
+import { SavedSetsContext } from 'providers/SavedSets';
+
+export default ({ modelName, sqon, className }) => (
+  <SavedSetsContext.Consumer>
+    {context => {
+      return (
+        <Component
+          name={modelName}
```

Comment:
cool! i didn't know you could add props like that. makes sense. like a little built in `mapProps`